apps/masters/views.py

```diff
@@ -1,20 +1,58 @@
-from django.shortcuts import render
-from rest_framework import viewsets
-from .models import *
-from .serializers import *
-from config.utils_methods import *
-from django_filters.rest_framework import DjangoFilterBackend
+from django_filters.rest_framework import DjangoFilterBackend # type: ignore
+from rest_framework.parsers import MultiPartParser, FormParser
 from rest_framework.filters import OrderingFilter
-from .filters import *
-from django.utils import timezone
 from rest_framework.decorators import api_view
 from rest_framework.response import Response
-from rest_framework import status
 from .models import generate_order_number
+from rest_framework.views import APIView
+from django.shortcuts import render
+from rest_framework import viewsets
+from config.utils_methods import *
+from rest_framework import status
+from django.conf import settings
+from .serializers import *
+from .filters import *
+from .models import *
+import json
+import os
```

Comment:
Can you avoid import * and import only needed ones from above import statements?

Reply:
They are importing all models, filters which are present in models.py and filters.py files